build.sh

```diff
@@ -358,6 +359,12 @@ if buildAll || hasArg cudf; then
         python ${PYTHON_ARGS_FOR_INSTALL} .
 fi
 
+# Build and install the cudf_polars Python package
+if buildAll || hasArg cudf_polars; then
+
+    cd ${REPODIR}/python/cudf_polars
+    python ${PYTHON_ARGS_FOR_INSTALL} .
```

Comment:
This doesn't itself install `polars` itself. Should `dependencies.yaml` be updated somehow such that polars is present the conda environments we often use to build from source, in the same way dask is present for instance?

Reply:
Yes, I did this in https://github.com/rapidsai/cudf/pull/16875. Merge that first, or merge it into this PR.

Reply:
ah, thanks @bdice . Let's get that one in and this one afterwards.